script.js

```diff
@@ -1,39 +1,60 @@
+var currentGameMode = "waiting for user name";
+var userName = "";
+var userScore = 0;
+var compScore = 0;
+var drawScore = 0;
+
 var main = function (input) {
-  var randomChoice = randomNumber();
-  var compChoice = "";
+  var myOutputValue = "";
 
-  //computer logic
-  if (randomChoice == 1) {
-    compChoice = "stone";
-  }
-  if (randomChoice == 2) {
-    compChoice = "paper";
-  }
-  if (randomChoice == 3) {
-    compChoice = "scissors";
-  }
```

Comment:
You could also put the part where randomChoice returns a word based on the number inside the helper function in order to make your own main function shorter. This can also eliminate the need for the compChoice variable.